sql/mysqld.cc

```diff
@@ -13027,42 +13027,31 @@ PSI_stage_info *all_server_stages[] = {
 #ifdef WITH_WSREP
 PSI_stage_info *wsrep_server_stages[] = {
     // log_event
-    &stage_wsrep_writing_rows,
-    &stage_wsrep_deleting_rows,
+    &stage_wsrep_writing_rows, &stage_wsrep_deleting_rows,
```

Comment:
This reformatting will cause the failure of rpl_perfschema_threads_processlist_status.test

Reply:
Fixed. Disabled `clang-format` check for this block and the test passes now.
```

==============================================================================
                  TEST NAME                       RESULT  TIME (ms) COMMENT
------------------------------------------------------------------------------
[ 20%] rpl.rpl_perfschema_threads_processlist_status 'mix'  [ skipped ]  Doesn't support --binlog-format = 'mixed'
[ 40%] rpl.rpl_perfschema_threads_processlist_status 'row'  [ pass ]    680
[ 60%] rpl.rpl_perfschema_threads_processlist_status 'stmt'  [ skipped ]  Doesn't support --binlog-format = 'statement'
[ 80%] rpl.rpl_perfschema_threads_processlist_status 'ENV'  [ pass ]    707
[100%] shutdown_report                           [ pass ]       
------------------------------------------------------------------------------
The servers were restarted 1 times
The servers were reinitialized 0 times
Spent 1.387 of 30 seconds executing testcases
```